src/main/java/io/github/wimdeblauwe/hsbt/mvc/HtmxHandlerInterceptor.java

```diff
@@ -0,0 +1,26 @@
+package io.github.wimdeblauwe.hsbt.mvc;
+
+import org.springframework.core.annotation.AnnotationUtils;
+import org.springframework.web.method.HandlerMethod;
+import org.springframework.web.servlet.HandlerInterceptor;
+import org.springframework.web.servlet.ModelAndView;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.lang.reflect.Method;
+
+public class HtmxHandlerInterceptor implements HandlerInterceptor {
+    @Override
+    public void postHandle(HttpServletRequest request,
+                           HttpServletResponse response,
+                           Object handler,
+                           ModelAndView modelAndView) throws Exception {
+        if (handler instanceof HandlerMethod) {
+            Method method = ((HandlerMethod) handler).getMethod();
+            HxTrigger methodAnnotation = AnnotationUtils.findAnnotation(method, HxTrigger.class);
```

Comment:
Will this method support subclassing (similar to how `@GetMapping` works)?

Reply:
Ah, that would be nice indeed. Let me write a test to find out if it does :)

Reply:
It did not work, but it was quickly easily fixed. Check the latest push on this branch to see more info.